src/filters/dasher.c

```diff
@@ -7458,7 +7468,9 @@ static void dasher_insert_timeline_entry(GF_DasherCtx *ctx, GF_DashStream *ds)
 	if (!s) return;
 	s->start_time = ds->seg_start_time + pto;
 	s->duration = (u32) duration;
+	if (ctx->dashll) s->nb_parts = sctx->nb_frags + 1;
```

Comment:
I have to add `sctx->nb_frags + 1` because there's always one more fragment left to be processed. Could be a bug.